### ISSUE TITLE
cryptonight-asmjs.min.js.mem

### ISSUE THREAD

**Issue by webmasterslava** [coin-hive-proxy-party/coin-hive-proxy-party.github.io#1]
Thanks for this great service!

This file
`cryptonight-asmjs.min.js.mem`
still loaded from
`https://coinhive.com/lib/cryptonight-asmjs.min.js.mem`
when "Force ASMJS" option is enabled.

Some Adblockers can recognize that and block mining.
```
GET https://coinhive.com/lib/cryptonight-asmjs.min.js.mem net::ERR_BLOCKED_BY_CLIENT
Uncaught could not load memory initializer https://coinhive.com/lib/cryptonight-asmjs.min.js.mem
```

**Comment by cazala** [comment]
nice catch, i ll fix it

**Comment by cazala** [comment]
I just released `v0.0.5` which fixes this issue. You can get a new proxy or rebuild your instance to get the changes.

**Comment by cazala** [comment]
<img width="624" alt="screen shot 2017-10-24 at 9 23 11 pm" src="https://user-images.githubusercontent.com/2781777/31974347-864d2a0c-b92b-11e7-949f-2f121570dab6.png">
